src/EditorFeatures/Core/Implementation/IntelliSense/SignatureHelp/Controller_InvokeSignatureHelp.cs

```diff
@@ -28,6 +28,11 @@ void IChainedCommandHandler<InvokeSignatureHelpCommandArgs>.ExecuteCommand(Invok
                 return;
             }
 
+            // Dismiss any completion sessions when Signature Help is explicitly invoked. The
+            // primary motivation for this behavior is when both Signature Help and Completion
+            // are activated at the same time, for example in argument lists.
+            _completionBroker.GetSession(args.TextView)?.Dismiss();
```

Comment:
nice!  glad this was so simple.  perhaps expand on the message to make it clear that there are cases where both come up *implciitly*, but this is the users was to *explicitly* state waht is important to them.

Reply:
Updated the comment.